mindsdb_native/libs/phases/model_analyzer/model_analyzer.py

```diff
@@ -45,6 +45,49 @@ def run(self):
         self.transaction.lmd['accuracy_samples'] = {}
         self.transaction.hmd['probabilistic_validators'] = {}
 
+
+        self.transaction.lmd['train_accuracy'] = {}
+        self.transaction.lmd['test_accuracy'] = {}
+        self.transaction.lmd['valid_accuracy'] = {}
```

Comment:
You need to modify `get_model_data` in order for these to be sent to `mindsdb` and then exposed via the http API